### ISSUE TITLE
[GraphOptz] Fold Exp + ReduceSum + Div into Softmax

### ISSUE THREAD

**Issue by mciprian13** [pytorch/glow#4654]
Some ONNX models contain this:
![image](https://user-images.githubusercontent.com/29785908/86492080-906b5d00-bd75-11ea-8cf2-724920acc503.png)
This should be folded into Softmax. The most important reason for this is the numerical stability since Exp, Div are numerically unstable whereas the Softmax is specially implemented/handled to avoid high values of the Exp.

**Comment by AhmedSamara** [comment]
Can you link a model that this occurs in?

**Comment by mciprian13** [comment]
@AhmedSamara I searched the model from where I extracted the attached picture but seems I cannot find it. Sorry!

**Comment by AhmedSamara** [comment]
@mciprian13 what is the model that it occurs in?
